src/chainstate/coordinator/mod.rs

```diff
@@ -427,8 +457,18 @@ pub fn get_reward_cycle_info<U: RewardSetProvider>(
                     sort_db,
                     &block_id,
                 )?;
+                test_debug!(
+                    "Stacks anchor block {}/{} is processed",
+                    &consensus_hash,
+                    &stacks_block_hash
+                );
                 PoxAnchorBlockStatus::SelectedAndKnown(stacks_block_hash, reward_set)
             } else {
+                test_debug!(
+                    "Stacks anchor block {}/{} is NOT processed",
+                    &consensus_hash,
+                    &stacks_block_hash
+                );
```

Comment:
Should these be `debug!` and `warn!` respectively rather than `test_debug!`?

Reply:
Done